dags/atd_knack_banner.py

```diff
@@ -17,7 +16,7 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "retries": 0,
-    "retry_delay": duration(minutes=5),
+    "execution_timeout": duration(minutes=30),
```

Comment:
Should this be longer if the whole DAG was timing out in 30 minutes? Ignore me if that was related to a bug that was making the runs longer than normal.

btw I found this little gem in the Airflow v3 docs which is not in our current version docs. Doesn't seem like it would be new behavior so probably them just improving their docs 🚧 

> dagrun_timeout – Specify the duration a DagRun should be allowed to run before it times out or fails. Task instances that are running when a DagRun is timed out will be marked as **skipped**.

Reply:
I was thinking about that, usually it shouldnt take more than a few minutes, but this is timing out because its trying to replace every record. I decided to keep it at 30 because if its taking longer than 30 that should mean something unexpected is happening